customer/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+
+from customer import views
+
+
+router = DefaultRouter()
+router.register('invoice', views.InvoiceViewSet)
```

Comment:
Please make it plural like so `router.register('invoices', views.InvoiceViewSet)`. We are following [JSONPlaceholder](https://jsonplaceholder.typicode.com/) REST URL naming convention.

Reply:
the endpoint should be `/invoices/` instead of `/invoice/`